config/initializers/whitelabel.rb

```diff
@@ -1,3 +1,5 @@
 Usergroup.initialize!
 
 I18n.extend WhitelabelTranslation
+
+Rails.application.config.middleware.use WhitelabelDetectionMiddleware
```

Comment:
I guess this need to be inserted before any omniauth rack middleware. So maybe do it explicitly?

Reply:
done